From 14f8fc7ff28995890d9d7654146e7a266c08fbaa Mon Sep 17 00:00:00 2001
From: WuTofu <5987870+WuTofu@users.noreply.github.com>
Date: Mon, 5 Jan 2026 14:05:01 +0800
Subject: [PATCH] openssl: only enable TLS1.3

---
 packages/openssl.cmake | 3 +++
 1 file changed, 3 insertions(+)

diff --git a/packages/openssl.cmake b/packages/openssl.cmake
index 523e93b..4e5febc 100644
--- a/packages/openssl.cmake
+++ b/packages/openssl.cmake
@@ -36,6 +36,9 @@ ExternalProject_Add(openssl
         no-apps
         no-ocsp
         no-ssl3
+        no-tls1
+        no-tls1_1
+        no-tls1_2
         no-cmac
         no-mdc2
         no-idea
-- 
2.43.0

